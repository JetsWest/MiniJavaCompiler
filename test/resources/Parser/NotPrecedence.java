class A {
    public static void main(String[] args) {
        System.out.println(new B().Not());
    }

}
class B {
    public boolean Not() {
        boolean a;
        a = false;
        c = !a;
        return c;
    }
}