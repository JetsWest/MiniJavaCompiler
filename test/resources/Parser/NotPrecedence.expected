Program
  MainClass A
    Print
      new B().Not()
  Class B
    MethodDecl Not
      returns boolean
      boolean a
      a = false
      c = !a
      Return c